tests/context_storages/test_dbs.py

```diff
@@ -121,42 +121,42 @@ def test_pickle(testing_file, testing_context, context_id):
     asyncio.run(delete_pickle(db))
 
 
+@pytest.mark.docker
 @pytest.mark.skipif(not MONGO_ACTIVE, reason="Mongodb server is not running")
 @pytest.mark.skipif(not mongo_available, reason="Mongodb dependencies missing")
-@pytest.mark.docker
```

Comment:
Why was the order changed?

Reply:
I might be wrong, but by the time I was writing this I was still unfamiliar with pytest custom annotations, so looking at telegram tests I decided that they (at least in our project) come before standard pytest annotations. I can revert it if necessary.